modular_skyrat/modules/stone/code/stone.dm

```diff
@@ -29,24 +29,42 @@ GLOBAL_LIST_INIT(stone_recipes, list ( \
 /datum/material/stone
 	name = "stone"
 	desc = "It's stone."
-	categories = list(MAT_CATEGORY_ORE = TRUE, MAT_CATEGORY_RIGID = TRUE, MAT_CATEGORY_BASE_RECIPES = TRUE, MAT_CATEGORY_ITEM_MATERIAL=TRUE)
+	categories = list(MAT_CATEGORY_RIGID = TRUE, MAT_CATEGORY_BASE_RECIPES = TRUE, MAT_CATEGORY_ITEM_MATERIAL=TRUE)
 	sheet_type = /obj/item/stack/sheet/mineral/stone
 	value_per_unit = 0.005
 	beauty_modifier = 0.01
+	color = "#59595a"
+	greyscale_colors = "#59595a"
+	value_per_unit = 0.0025
+	armor_modifiers = list(MELEE = 0.75, BULLET = 0.5, LASER = 1.25, ENERGY = 0.5, BOMB = 0.5, BIO = 0.25, FIRE = 1.5, ACID = 1.5)
+	beauty_modifier = 0.3
+	turf_sound_override = FOOTSTEP_PLATING
 
-/obj/item/stack/ore/stone
-	name = "unrefined stone"
-	desc = "Chunks of high-quality stone. This could be cut into bricks to form a really good building material."
+/obj/item/stack/stone
+	name = "rough stone"
+	desc = "Large chunks of uncut stone, tough enough to safely build out of... if you could manage to cut them into something usable."
 	icon = 'modular_skyrat/modules/stone/icons/ore.dmi'
 	icon_state = "stone_ore"
-	singular_name = "unrefined stone chunk"
-	points = 1
+	singular_name = "rough stone boulder"
 	mats_per_unit = list(/datum/material/stone=MINERAL_MATERIAL_AMOUNT)
-	refined_type = /obj/item/stack/sheet/mineral/stone
-	mine_experience = 1
-	scan_state = null
-	spreadChance = 20
-	merge_type = /obj/item/stack/ore/stone
+	merge_type = /obj/item/stack/stone
+
+/obj/item/stack/stone/examine()
+	. = ..()
+	. += span_notice("With a <b>chisel</b> or even a <b>pickaxe</b> of some kind, you could cut this into <b>blocks</b>.")
+
+/obj/item/stack/stone/attackby(obj/item/attacking_item, mob/user, params)
+	if(!(attacking_item.tool_behaviour != TOOL_MINING) || !(istype(attacking_item, /obj/item/chisel)))
```

Comment:
The first half has a double-negative, `!(x != y)`, so it would return if it IS a mining tool. I don't think you meant to have that first `!`.

```suggestion
	if((attacking_item.tool_behaviour != TOOL_MINING) || !(istype(attacking_item, /obj/item/chisel)))
```